src/gdft.c

```diff
@@ -35,6 +35,12 @@
 #ifndef R_OK
 #define R_OK 04			/* Needed in Windows */
 #endif
+
+#define ssize_t int
```

Comment:
This looks like a bad idea. If we really want to support old compilers, we should define `ssize_t` only if it's not available. And I would also suggest to use a type with the proper size (i.e. `long long` or so for x64).

Reply:
Thanks suggest. I have fix it again.